queries/leo/indents.scm

```diff
@@ -0,0 +1,33 @@
+[
+ (record_declaration) 
+ (struct_declaration)
+ (mapping_declaration)
+ (constant_declaration)
+ (return_statement)
+ (expression_statement)
+ (variable_declaration)
+ (loop_statement)
+ (assignment_statement)
+ (assert_statement)
+ (struct_expression)
+ (array_expression)
+ (tuple_expression)
+ (parenthesized_expression)
+ (items_block)
+ (block)
+] @indent.begin
+
+; if "if" statement and conditional statement are in separate lines 
+; conditional should be indented and when the conditional block 
+; starts dedented
```

Comment:
Does this mean the tree is like

```
branch
  block
   branch
    block
```

Reply:
Ah I understand now.
Can you make the condition of the if statement be a singular node, and if not, can you explain why

Something like

```
if_statement
  condition: (_)
  consequence: (block)
```

Reply:
No, I do not do it to indent the block, the condition after "if" statement must be indented if written in a new line. And it  was hard first as it has no opening and closing parens as C does, so the block opening brace will end the indent.